packages/e2e-test-app-fabric/jest.config.js

```diff
@@ -10,11 +10,6 @@
 // https://jestjs.io/docs/en/configuration.html
```

Comment:
Should we consolidate `jest.config.js` and `jest.setup.js`? They look mostly the same to me.